test/integration/connection-test.js

```diff
@@ -0,0 +1,1265 @@
+    test.done();
+  });
+
+  connection.on('infoMessage', function(info) {
+    //console.log("#{info.number} : #{info.message}")
+  });
+
+  connection.on('debug', function(text) {
+    //console.log(text)
+  });
+};
+
+exports.execSqlMultipleTimes = function(test) {
+  var timesToExec = 5;
+  var sqlExecCount = 0;
+
+  test.expect(timesToExec * 7);
+
+  var config = getConfig();
+
+  var execSql = function() {
+    if (sqlExecCount === timesToExec) {
+      connection.close();
+      return;
+    }
+
+    var request = new Request('select 8 as C1', function(err, rowCount) {
+      test.ifError(err);
+      test.strictEqual(rowCount, 1);
+
+      sqlExecCount++;
+      execSql();
+    });
+
+    request.on('doneInProc', function(rowCount, more) {
+      test.ok(more);
+      test.strictEqual(rowCount, 1);
+    });
+
+    request.on('columnMetadata', function(columnsMetadata) {
+      test.strictEqual(columnsMetadata.length, 1);
+    });
+
+    request.on('row', function(columns) {
+      test.strictEqual(columns.length, 1);
+      test.strictEqual(columns[0].value, 8);
+    });
+
+    connection.execSql(request);
+  };
+
+  var connection = new Connection(config);
+
+  connection.on('connect', function(err) {
+    execSql();
+  });
+
+  connection.on('end', function(info) {
+    test.done();
+  });
+
+  connection.on('infoMessage', function(info) {
+    //console.log("#{info.number} : #{info.message}")
+  });
+
+  connection.on('debug', function(text) {
+    //console.log(text)
+  });
+};
+
+exports.execSqlWithOrder = function(test) {
+  test.expect(10);
+
+  var config = getConfig();
+
+  var sql =
+    'select top 2 object_id, name, column_id, system_type_id from sys.columns order by name, system_type_id';
+  var request = new Request(sql, function(err, rowCount) {
+    test.ifError(err);
+    test.strictEqual(rowCount, 2);
+
+    connection.close();
+  });
+
+  request.on('doneInProc', function(rowCount, more) {
+    test.ok(more);
+    test.strictEqual(rowCount, 2);
+  });
+
+  request.on('columnMetadata', function(columnsMetadata) {
+    test.strictEqual(columnsMetadata.length, 4);
+  });
+
+  request.on('order', function(orderColumns) {
+    test.strictEqual(orderColumns.length, 2);
+    test.strictEqual(orderColumns[0], 2);
+    test.strictEqual(orderColumns[1], 4);
+  });
+
+  request.on('row', function(columns) {
+    test.strictEqual(columns.length, 4);
+  });
+
+  var connection = new Connection(config);
+
+  connection.on('connect', function(err) {
+    connection.execSql(request);
+  });
+
+  connection.on('end', function(info) {
+    test.done();
+  });
+
+  connection.on('infoMessage', function(info) {
+    //console.log("#{info.number} : #{info.message}")
+  });
+
+  connection.on('errorMessage', function(error) {
+    //console.log("#{error.number} : #{error.message}")
+  });
+
+  connection.on('debug', function(text) {
+    //console.log(text)
+  });
+};
+
+exports.execSqlMultipleTimes2 = function(test) {
```

Comment:
You can remove this test. This looks like a less complete duplicate of execSqlMultipleTimes. In the .coffee file the same name is used twice, so one of them is overriding the other. The conversion tool seems to have renamed the second instance.